airgun/views/acs.py

```diff
@@ -0,0 +1,447 @@
+import time
+
+from widgetastic.widget import Checkbox
+from widgetastic.widget import Text
+from widgetastic.widget import TextInput
+from widgetastic.widget import View
+from widgetastic_patternfly4 import Button
+from widgetastic_patternfly4 import Drawer
+from widgetastic_patternfly4 import Dropdown
+from widgetastic_patternfly4 import FormSelect
+from widgetastic_patternfly4 import Pagination
+from widgetastic_patternfly4 import Radio
+from widgetastic_patternfly4 import Switch
+from widgetastic_patternfly4.ouia import Button as OUIAButton
+from widgetastic_patternfly4.ouia import FormSelect as OUIAFormSelect
+from widgetastic_patternfly4.ouia import PatternflyTable
+from widgetastic_patternfly4.ouia import Switch as OUIASwitch
+from widgetastic_patternfly4.ouia import Text as OUIAText
+from widgetastic_patternfly4.ouia import TextInput as OUIATextInput
+
+from airgun.views.common import BaseLoggedInView
+from airgun.views.common import WizardStepView
+from airgun.views.host_new import SearchInput
+from airgun.widgets import ItemsList
+
+
+class EditModal(View):
+    """Class representing the Edit modal header"""
+
+    title = Text('.//h1')
+    close_button = OUIAButton('acs-edit-details-modal-ModalBoxCloseButton')
+
+    error_message = Text('//div[contains(@aria-label, "Danger Alert")]')
```

Comment:
edit_button, cancel_button can be also included here, move this class to the common widget.py. So this can be further used for other entities